server/models/pattern/patterns/patterns.go

```diff
@@ -18,7 +17,21 @@ import (
 	"github.com/spf13/viper"
 )
 
-func ProcessOAM(kconfigs []string, oamComps []string, oamConfig string, isDel bool, patternName string, ec *models.Broadcast, userID string, provider models.Provider, hostname v1beta1.IHost, skipCrdAndOperator bool) (string, error) {
+type DeploymentMessagePerComp struct {
+	Kind       string
+	CompName   string
+	Success    bool
+	DesignName string
+	Message    string
+}
+
+type DeploymentMessagePerContext struct {
+	Summary    []DeploymentMessagePerComp
+	ServerName string
```

Comment:
Is "ServerName" defined in the Events schema?

Reply:
resolved in slack,